dev/perf/scheduler_dag_execution_timing.py

```diff
@@ -79,8 +79,7 @@ def change_state(self, key, state, info=None):
         if not run:
             import airflow.models
 
-            # odd `list()` is to work across Airflow versions.
-            run = list(airflow.models.DagRun.find(dag_id=dag_id, execution_date=execution_date))[0]
+            run = airflow.models.DagRun.find(dag_id=dag_id, execution_date=execution_date)[0]
```

Comment:
This comment seems to suggest `find` can return something other than a list instead…? But that’s four years ago so it could be fine to remove it.

Reply:
yes. It was for 1.10 I believe.